sdk/testing/helpers/repo.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // RepositoryDir returns the absolute path of kcp-dev/kcp repository on disk.
-func RepositoryDir() string {
+func RepositoryDir() (string, error) {
```

Comment:
The workdir has to be passed, otherwise `go run` tries to use the working directories `go.mod` to run kcp, instead of the `go.mod` of the kcp clone.